android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeModule.kt

```diff
@@ -16,6 +16,14 @@ class CustomerIOReactNativeModule(
         get() = CustomerIOShared.instance().diStaticGraph.logger
     private lateinit var customerIO: CustomerIO
 
+    init {
+        // If SDK is already initialized from CIO service using context, initialize the local
+        // reference with SDK instance
```

Comment:
This comment explains what the code is doing, but it doesn't explain why we are initializing the local SDK reference. What problem this code solves. 

A suggestion is that the comment could be modified to answer the question: 

Why do we have this code instead of modifying the class from:

```kt
private lateinit var customerIO: CustomerIO
```

to:

```kt
private val customerIO = CustomerIO.instance()
```